deployments/apiv2/services/users/src/main.py

```diff
@@ -1,3 +1,4 @@
+import datetime
```

Comment:
We also want to make sure that the new routes in this file have tests written for them